java/ql/src/experimental/CWE-939/IncorrectURLVerification.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @id java/incorrect-url-verification
+ * @name Insertion of sensitive information into log files
+ * @description Apps that rely on URL parsing to verify that a given URL is pointing to a trusted server are susceptible to wrong ways of URL parsing and verification. 
+ * @kind problem
+ * @tags security
+ *       external/cwe-939
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+import DataFlow
+import PathGraph
```

Comment:
These imports do not appear to be needed.

Reply:
Thanks @aschackmull . You are right, those imports are not needed thus have been removed from the new commit I just made.

@luchua-bc